config/application.rb

```diff
@@ -42,5 +42,14 @@ class Application < Rails::Application
 
     # Permitted locales available for the application
     config.i18n.available_locales = [:en, :cn]
+
+    config_files = ['secrets.yml']
```

Comment:
I don't think we have a `secrets.yml`?

Reply:
Not at all. And it upset me for a while. Then I added it, and then I had an aha moment and realized that secrets.yml isn't used anymore. Then deleted it, but some traces were left behind.

Reply:
😆